src/MahApps.Metro/Styles/Themes/GeneratorParameters.json

```diff
@@ -91,7 +179,94 @@
+        "MahApps.Colors.Mui.LinkButtonTextHover": "#333333",
+        "MahApps.Colors.Mui.LinkButtonTextPressed": "#919191",
+        "MahApps.Colors.Mui.LinkButtonTextDisabled": "#919191",
+        "MahApps.Colors.Mui.MenuText": "#66000000",
+        "MahApps.Colors.Mui.MenuTextHover": "#bb000000",
+        "MahApps.Colors.Mui.MenuTextSelected": "#333333",
+        "MahApps.Colors.Mui.ModernButtonBorder": "#919191",
+        "MahApps.Colors.Mui.ModernButtonBorderHover": "#333333",
+        "MahApps.Colors.Mui.ModernButtonBorderPressed": "#333333",
+        "MahApps.Colors.Mui.ModernButtonBorderDisabled": "#919191",
+        "MahApps.Colors.Mui.ModernButtonIconForegroundPressed": "#ffffff",
+        "MahApps.Colors.Mui.ModernButtonText": "#333333",
+        "MahApps.Colors.Mui.ModernButtonTextHover": "#333333",
+        "MahApps.Colors.Mui.ModernButtonTextPressed": "#333333",
+        "MahApps.Colors.Mui.ModernButtonTextDisabled": "#a1a1a1",
+        "MahApps.Colors.Mui.PopupBackground": "#ffffff",
+        "MahApps.Colors.Mui.ProgressBackground": "#dddddd",
+        "MahApps.Colors.Mui.ScrollBarBackground": "Transparent",
+        "MahApps.Colors.Mui.ScrollBarThumbBackground": "Transparent",
+        "MahApps.Colors.Mui.ScrollBarThumbBackgroundDragging": "#cccccc",
+        "MahApps.Colors.Mui.ScrollBarThumbBackgroundHover": "#dddddd",
+        "MahApps.Colors.Mui.ScrollBarThumbBorder": "#cccccc",
+        "MahApps.Colors.Mui.ScrollBarThumbForeground": "Transparent",
+        "MahApps.Colors.Mui.ScrollBarThumbForegroundDragging": "Transparent",
+        "MahApps.Colors.Mui.ScrollBarThumbForegroundHover": "Transparent",
+        "MahApps.Colors.Mui.SeparatorBackground": "#cccccc",
+        "MahApps.Colors.Mui.SliderThumbBackground": "#ffffff",
+        "MahApps.Colors.Mui.SliderThumbBackgroundHover": "#dddddd",
+        "MahApps.Colors.Mui.SliderThumbBackgroundDisabled": "#cccccc",
+        "MahApps.Colors.Mui.SliderThumbBorder": "#cccccc",
+        "MahApps.Colors.Mui.SliderThumbBorderHover": "#cccccc",
+        "MahApps.Colors.Mui.SliderThumbBorderDisabled": "#cccccc",
+        "MahApps.Colors.Mui.SliderTrackBackground": "#cccccc",
+        "MahApps.Colors.Mui.SliderTrackBorder": "#cccccc",
+        "MahApps.Colors.Mui.SliderTick": "#919191",
+        "MahApps.Colors.Mui.SliderTickDisabled": "#cccccc",
+        "MahApps.Colors.Mui.SubMenuText": "#99000000",
+        "MahApps.Colors.Mui.SubMenuTextHover": "#66000000",
+        "MahApps.Colors.Mui.SubMenuTextSelected": "#333333",
+        "MahApps.Colors.Mui.WindowHeaderGradient": "<LinearGradientBrush x:Key=\"MahApps.Brushes.Mui.WindowHeaderGradient\" StartPoint=\"0, 0\" EndPoint=\"0, 1\" Opacity=\".1\">\r\n<GradientStop Offset=\"0\" Color=\"{DynamicResource MahApps.Colors.Mui.AccentColor}\" />\r\n<GradientStop Offset=\".3\" Color=\"{DynamicResource MahApps.Colors.Mui.AccentColor}\" />\r\n<GradientStop Offset=\"1\" Color=\"Transparent\" />\r\n</LinearGradientBrush>",
```

Comment:
Dark and White theme not only changes the color but uses completely different brushes so I hacked the json generator to generate the full brush. I am open to better suggestions.

Reply:
Is it ok for you @punker76?